src/LayerPlan.cpp

```diff
@@ -1463,7 +1473,8 @@ void LayerPlan::writeGCode(GCodeExport& gcode)
                 gcode.writeRetraction(retraction_config);
                 if (path.perform_z_hop)
                 {
-                    gcode.writeZhopStart(retraction_config.zHop);
+                    gcode.writeZhopStart(z_hop_height);
+                    z_hop_height = retraction_config.zHop; // back to normal z hop
```

Comment:
It could be that I'm missing something, but I don't get this line: `z_hop_height` is not a reference, is not used below this line and is defined _within_ the scope of the enclosing for loop, so what effect is this supposed to have?

Reply:
Yes, this is redundant. Thanks! Fixed.